package.json

```diff
@@ -59,7 +60,7 @@
     "intl-messageformat": "1.3.0",
     "jsonpath": "0.2.9",
     "knex": "0.12.5",
-    "knex-migrator": "2.0.7",
+    "knex-migrator": "^2.0.7",
```

Comment:
Could this change be removed please? We like to keep our dependencies pinned at a specific version. Thanks!